extensions/parquet/base/src/test/java/io/deephaven/parquet/base/util/CachedChannelProviderTest.java

```diff
@@ -178,6 +192,9 @@ public SeekableByteChannel getWriteChannel(@NotNull String path, boolean append)
         public SeekableByteChannel getWriteChannel(@NotNull Path path, boolean append) {
             return new TestMockChannel(count.getAndIncrement(), path.toString());
         }
+
+        @Override
+        public void close() {}
     }
 
     private class TestMockChannel implements SeekableByteChannel {
```

Comment:
It might be worth it to update this to be a `ContextHolder` and verify that it gets a context when it should, and gets cleared when it should.